app/src/main/kotlin/co/netguru/baby/monitor/client/feature/analytics/AnalyticsManager.kt

```diff
@@ -0,0 +1,69 @@
+package co.netguru.baby.monitor.client.feature.analytics
+
+import android.app.Activity
+import androidx.core.os.bundleOf
+import com.google.firebase.analytics.FirebaseAnalytics
+import timber.log.Timber
+
+class AnalyticsManager(
+    private val firebaseAnalytics: FirebaseAnalytics
+) {
+    fun setCurrentScreen(activity: Activity, screenName: String) {
+        firebaseAnalytics.setCurrentScreen(activity, screenName, null)
+        Timber.d("$SCREEN $screenName")
+    }
+
+    fun logEvent(eventName: String) {
+        firebaseAnalytics.logEvent(eventName, null)
+        Timber.d("$EVENT $eventName")
+    }
+
+    fun logEventWithParam(eventName: String, pair: Pair<String, Any>) {
+        firebaseAnalytics.logEvent(eventName, bundleOf(pair))
+        Timber.d("$EVENT $eventName")
+    }
```

Comment:
You could refactor these methods into a single one, with an optional parameter defaulting to null.

Reply:
I would think about typing `eventName` argument with an `enum` if you don't require arbitrary strings to be permitted?